RatersOfTheLostBusiness/RatersOfTheLostBusiness/Data/BusinessDbContext.cs

```diff
@@ -26,19 +26,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
 
-            //id, name, city, state, address, phone, type
+            // Businesses
+            // id, name, city, state, address, phone, type
             modelBuilder.Entity<Business>().HasData(
                 new Business
                 {
-                    Id = 1,
-                    Name = "Twilio",
-                    Address = "375 Beale Street Suite 300",
-                    City = "San Franciso",
-                    State = "CA",
-                    PhoneNumber = "844-814-4627",
-                    Type = "Software Service"
+                    Id = 1, Name = "Twilio", Address = "375 Beale Street Suite 300", City = "San Franciso", State = "CA", PhoneNumber = "844-814-4627", Type = "Software Service"

```

Comment:
new Object constructor conventions were inconsistent. Some were written inline, and others in rows.